generator.rb

```diff
@@ -0,0 +1,63 @@
+# method to determine if letter is a value
+def is_vowel?(letter)
+  if ["a", "e", "i", "o", "u"].include?(letter)
+    return true
+  else return false   end
+end
+
+def run_generator
+  # Gets input from user for first word, will not accept fewer than two letters
+  puts "What's the first word you would like to combine?: "
+  word_a = gets.chomp.downcase
+  until word_a.length >= 2
+    puts "Your word must be 2 characters or longer."
+    puts "Please try again and enter your first word: "
+    word_a = gets.chomp.downcase
+  end
+  # Gets input from user for second word, will not accept fewer than two letters
+  puts "What's the second word you would like to combine?: "
+  word_b = gets.chomp.downcase
+  until word_b.length >= 2
+    puts "You word must be 2 characters or longer."
+    puts "Please try again and enter your second word: "
+    word_b = gets.chomp.downcase
+  end
+  puts "The first word is #{word_a}"
```

Comment:
One general changed I'd look for in your code is more whitespace! It's good to group code by adding blank lines so that we can tell what sections represent a "single thought". Convention in most workspaces is to have whitespace before and/or after loops and method declarations.